janusgraph-all/src/test/java/org/janusgraph/core/EnsureCacheTest.java

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph.core;
+
+import org.apache.commons.configuration.MapConfiguration;
+import org.janusgraph.diskstorage.configuration.backend.CommonsConfiguration;
+import org.janusgraph.graphdb.configuration.builder.GraphDatabaseConfigurationBuilder;
+import org.janusgraph.graphdb.database.StandardJanusGraph;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.janusgraph.graphdb.configuration.GraphDatabaseConfiguration.STORAGE_BACKEND;
+
+public class EnsureCacheTest {
```

Comment:
From your comment in the PR, I can see you added this test so that test dependencies can be cached in GitHub workflow. Can you add Javadoc here to illustrate it? By simply looking at the code itself, one might be confused.

Reply:
Agree here. It's not obvious what this test does. It's better to add some comments/javadoc to explain it